dashboard/test/ui/runner.rb

```diff
@@ -527,7 +532,7 @@ def how_many_reruns?(test_run_string)
   FileUtils.rm rerun_filename, force: true
 
   reruns = 0
-  succeeded, output_stdout, output_stderr, test_duration = run_tests(run_environment, arguments, log_prefix)
+  succeeded, output_stdout, output_stderr, test_duration = run_tests(run_environment, feature, arguments, log_prefix)
   log_link = upload_log_and_get_public_link(
```

Comment:
:exclamation: Heads up: I gave this a try locally, and I get errors here when we try to upload the log to S3 because now that we skip features properly, no log is generated in some cases. Worse, in some environments it might succeed with an old log.

```
[development] Uploading log to S3 failed: No such file or directory @ rb_sysopen - iPhone_applab_scenarios_eyes_output.html
```

We could make `upload_log_and_get_public_link` check that the file exists first - maybe a good solution for now - or do a more complex refactor to pull that step inside the check for whether we should run anything.